build-tools/packages/build-tools/src/fluidBuild/tasks/taskFactory.ts

```diff
@@ -63,6 +63,15 @@ const executableToLeafTask: {
 	"flub rename-types": RenameTypesTask,
 };
 
+/**
+ * Regular expression to parse `concurrently` arguments that specify package scripts.
+ * The format is `npm:<script>` or `"npm:<script>*"`; in the latter case script
```

Comment:
Does this match `concurrently`'s spec? This seems like the most common use-case, but I'm curious if something like `build:*-tests` is supported too?

If they support general glob patterns, it might be better for us to use something like [minimatch](https://www.npmjs.com/package/minimatch) for glob resolution.

Reply:
`concurrently` has a lot of things that fluid-build doesn't support. The docs I found don't appear to be complete. There is support for an exclusion notation as well. I think we can keep it simple. We don't want to make it hard to understand what will be done; so, fluid-build doesn't need to support complicated things. <prefix>* is a common pattern in the code base already and I think well-understood. I think we stay with that.

Reply:
I think \* is enough for now, and in general I think build-tools is a place where we can be minimalist (where minimalist in this case is "we build only what we need").

Reply:
I would argue that using a more general glob resolution tool to resolve glob patterns is more minimalist than adding our own custom regex_s that are more fragile. Not blocking on this, but it seems like a pretty easy win IMO.